lib/screens/account.dart

```diff
@@ -21,34 +42,58 @@ class Account extends StatelessWidget {
       body: Container(
         child: accountOptions(),
       ),
+      drawer: HomeDrawer(),
     );
   }
 
+  getDate() async {
+    final SharedPreferences prefs = await SharedPreferences.getInstance();
+    String createdAt = prefs.getString('createdAt');
+    setState(() {
+      createdAtString = formatter.format(DateTime.parse(createdAt));
+    });
+  }
+
   Widget accountOptions() {
     return ScopedModelDescendant(
         builder: (BuildContext context, Widget child, MainModel model) {
       return ListView(
         children: <Widget>[
+          Container(
+            color: Colors.grey.shade100,
+            child: ListTile(
+              title: Text(
+                "Customer since $createdAtString",
+                style: TextStyle(
+                    color: Colors.grey,
+                    fontSize: 12,
+                    fontWeight: FontWeight.w600),
+              ),
+              onTap: () {},
```

Comment:
no method call for onTap().

Reply:
This is just info about the user, doesn't require to lead somewhere onTap, that's why I have kept it empty.